### DISCUSSION TITLE
Idea: BOX86 bare metal Hypervisor.

### DISCUSSION THREAD

**Discussion by Lander1979** [ptitSeb/box86#463] [Ideas]
Hi, I am a fairly novice Linux user, so far I have been experimenting and failing miserably at attempting to use x86 linux software on the Pi4b as the flatpack managers and apt repositories are focused on not letting me download or install anything marked as i386 because I am on ARM.  So far any attempt I've made to "mix distro's" has resulted in me just breaking packages with irreconsilable dependancies.

I was thinking, it would be great to have BOX86 as a hypervisor and just install whatever i386 distro on top, negating the dependancy hell between ARMHF and i386 all together.

**Comment by ptitSeb** [comment]
That's really not the initial purpose of box86. box86 rely a lot on the system on wich it is running. And box86 doesn't implement most of the needed low-level opcode and function needed to run a distro.

Flatpak/apt will not really be plug'n play with box86. But box86 can be used easily to run software / games you can get in a tar.gz archive. Just untar the archive and launch the binary. This should work most of the time!
